README.md

```diff
@@ -10,30 +10,28 @@ Sacrificial VM provides infrastructure for containers.
 
 ### Ports
 
-SSH:
-- Gateway VM SSH Honeypot: `2222`, `22`
-- Gateway VM SSHD: `2333`
-- Other VMs SSHD: `22`
-
-Audit:
+Gateway VM:
+- Honeypot gateway: `22`, `2222` (`2222` is redirected to `22`)
+- SSH: `2333`
+- cAdvisor: `8088`
+- Node Exporter: `9100`
+- ContainerSSH auth-config server: `8080`
```

Comment:
removed `auth-config` from the infra diagram since it might be not very important to our project.